aws-iot-accountauditconfiguration/src/test/java/com/amazonaws/iot/accountauditconfiguration/TranslatorTest.java

```diff
@@ -47,6 +48,13 @@ void translateChecksFromIotToCfn_NonNull_VerifyTranslation() {
         assertThat(Translator.translateChecksFromIotToCfn(input)).isEqualTo(DESCRIBE_RESPONSE_V1_STATE_CFN);
     }
 
+    @Test
+    void translateChecksFromIotToCfn_NonExistentKeyFromIot_VerifyIgnoredInTranslation() {
```

Comment:
Let's add some more tests:
1. translate**Notifications**FromIotToCfn_NonExistentKeyFromIot_VerifyIgnoredInTranslation
2. translate**Checks**FromCfnToIot_NullIn_EmptyOut
3. translateChecksFromCfnToIot_AllChecksArePresent - to cover every check and code branch
4. translateChecksFromIotToCfn_AllChecksArePresent - same but in the other direction

Reply:
Done